app/components/personalcircle.js

```diff
@@ -1,18 +1,145 @@
-import { Button, ButtonGroup } from "react-bootstrap";
-import styles from '../styles/Personacircle.module.css'
+import { Button, ButtonGroup, Col } from 'react-bootstrap';
+import styles from '../styles/Personacircle.module.css';
 
 export default function Personacircle(props) {
-    return (
-        <>
-        <ButtonGroup size="lg" className="mb-2">
-        <span className={styles["select-role-buttons-row"]}>
-          {props.personas.map((persona) => (
-            <span className={styles["select-role-button"]}>
-              <Button>{persona.name}</Button>
-            </span>
-          ))}
+  return (
+    <>
+      <Col
+        className={`${styles.personas} d-flex flex-wrap justify-content-center`}
+      >
+        <span className={`${styles.persona}`}>
+          <div className={styles.svg}>
```

Comment:
pls move in-line svg as per earlier comment.

Reply:
In the previous code, we were getting all the data used here ( like name and icons ) from backend, so we can replace the svg's there with these svg's directly and change the frontend code when that is done